### PR TITLE
update readme

### PR DESCRIPTION
**Pull request by JulianVennen**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -12,22 +12,23 @@ Modbot is a Moderation Bot specifically for the [Aternos Discord](https://chat.a
...
+4. create a discord Application [here](https://discordapp.com/developers/applications/)
+5. add a Bot to it and copy the auth token
+6. copy the example.config.json to config.json and enter the auth token and mysql login
+7. run this command in the directory
 
    ```bash
    node bot.js
    ```
-7. to invite the bot to your server replace ID with your applications Client ID https://discordapp.com/oauth2/authorize?client_id=ID&scope=bot&permissions=11264 and open the link
+8. to invite the bot to your server replace ID with your applications Client ID https://discordapp.com/oauth2/authorize?client_id=ID&scope=bot&permissions=11264 and open the link
```
Comment (matthi4s) [inline README.md:26]:
```suggestion
8. To invite the bot to your server replace `ID` with the client ID of your application https://discordapp.com/oauth2/authorize?client_id=ID&scope=bot&permissions=11264 and open the link
```

File: README.md [hunk 2]
```diff
@@ -12,22 +12,23 @@ Modbot is a Moderation Bot specifically for the [Aternos Discord](https://chat.a
...
 
-8. (optional) Set up a logchannel by using ,logchannel #channel
+9. (optional) Set up a logchannel by using ,logchannel #channel
```
Comment (matthi4s) [inline README.md:28]:
```suggestion
9. (optional) Set up a log channel by using !logchannel #channel
```

File: README.md [hunk 3]
```diff
@@ -12,22 +12,23 @@ Modbot is a Moderation Bot specifically for the [Aternos Discord](https://chat.a
...
 
-1. Install [Node.js](https://nodejs.org/en/)
-2. Clone or download the repository
-3. create a discord Application [here](https://discordapp.com/developers/applications/)
-4. add a Bot to it and copy the auth token
-5. rename the example.config.json to config.json and enter the auth token
-6. run this command in the directory
+1. Install [Node.js](https://nodejs.org/en/) and [Mysql](https://www.mysql.com/)
+2. create a user and databse for the bot
+3. Clone or download the repository
```
Comment (matthi4s) [inline README.md:17]: Add `npm install` as a step

**Review by matthi4s — CHANGES_REQUESTED** [review]

File: README.md [hunk 4]
```diff
@@ -12,22 +12,24 @@ Modbot is a Moderation Bot specifically for the [Aternos Discord](https://chat.a
...
-1. Install [Node.js](https://nodejs.org/en/)
-2. Clone or download the repository
-3. create a discord Application [here](https://discordapp.com/developers/applications/)
-4. add a Bot to it and copy the auth token
-5. rename the example.config.json to config.json and enter the auth token
-6. run this command in the directory
+1. Install [Node.js](https://nodejs.org/en/download/) and [MySQL](https://dev.mysql.com/downloads/mysql/)
+2. Create a MySQL user and database for the bot
+3. Clone or download the repository
+4. run `npm install`
```
Comment (matthi4s) [inline README.md:18]:
```suggestion
4. Run `npm install`
```

**Review by matthi4s — CHANGES_REQUESTED** [review]

**Review by matthi4s — APPROVED** [review]

**Review by KurtThiemann — APPROVED** [review]

**Pull request merged**